### PR TITLE
Add docker file and rework appsettings

### PR DESCRIPTION
**Pull request by djino97**

### CODE REVIEW

File: src/ProjectService/appsettings.Production.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "Logging": {
```
Comment (DarkFelas) [inline src/ProjectService/appsettings.Production.json:2]: remove Logging section

File: src/ProjectService/appsettings.Development.json [hunk 2]
```diff
@@ -5,5 +5,11 @@
       "Microsoft": "Warning",
```
Comment (DarkFelas) [inline src/ProjectService/appsettings.Development.json:5]: remove Logging section

File: src/ProjectService/appsettings.json [hunk 3]
```diff
@@ -24,13 +24,14 @@
...
     "Username": "ProjectService",
     "Password": "E68A3452-AEAF-4E28-92D5-CCC5EE9F0A4C",
-    "AccessValidatorCheckRightsServiceURL": "rabbitmq://localhost/CheckRightsService",
-    "AccessValidatorUserServiceURL": "rabbitmq://localhost/UserService",
-    "FileServiceUrl": "rabbitmq://localhost/FileService",
-    "UserServiceUsersUrl": "rabbitmq://localhost/UserService",
-    "CompanyServiceDepartmentsUrl": "rabbitmq://localhost/CompanyService_Departments"
+
+    // Request urls
+    "AccessValidatorCheckRightsServiceURL": "CheckRightsService",
```
Comment (DarkFelas) [inline src/ProjectService/appsettings.json:31]: Check example in UserService

**Review by DarkFelas — CHANGES_REQUESTED** [review]

File: src/ProjectService/appsettings.Production.json (hunk 1 above)
Comment (djino97) [reply src/ProjectService/appsettings.Production.json]: Fixed

File: src/ProjectService/appsettings.Development.json (hunk 2 above)
Comment (djino97) [reply src/ProjectService/appsettings.Development.json]: Fixed

File: src/ProjectService/appsettings.json (hunk 3 above)
Comment (djino97) [reply src/ProjectService/appsettings.json]: Renamed endpoints and values. Fixed.

**Review by DarkFelas — APPROVED** [review]

**Pull request merged**